### ISSUE TITLE
Docker  18.06 and 18.09 support to rke v0.1.x line

### ISSUE THREAD

**Issue by alena1108** [rancher/rke#1132]
Enable 18.09.x docker support to align with Rancher SLA https://rancher.com/support-maintenance-terms/

**Comment by deniseschannon** [comment]
Let's also add in 18.06.2.

**Comment by alena1108** [comment]
Available in `v0.1.16-rc1`

**Comment by soumyalj** [comment]
Reproduced the unsupported docker versions error for docker versions 18.06 and 18.09
```
soumyas-MacBook-Pro:rke soumya$ ./rke_darwin-amd64 up --config cluster.yml 
INFO[0000] Building Kubernetes cluster                  
INFO[0000] [dialer] Setup tunnel for host [a.b.c.d] 
FATA[0000] Unsupported Docker version found [18.06.2-ce], supported versions are [1.11.x 1.12.x 1.13.x 17.03.x] 

soumyas-MacBook-Pro:rke soumya$ ./rke_darwin-amd64 up --config cluster.yml 
WARN[0000] This is not an officially supported version (v0.1.16-rc1) of RKE. Please download the latest official release at https://github.com/rancher/rke/releases/latest 
INFO[0000] Building Kubernetes cluster                  
INFO[0000] [dialer] Setup tunnel for host [a.b.c.d] 
FATA[0000] Unsupported Docker version found [18.09.2], supported versions are [1.11.x 1.12.x 1.13.x 17.03.x] 


```
Tested with rke version v0.1.16-rc1 with K8s version v1.12.4-rancher1-1
1. Created DO nodes with docker version 18.06.2-ce. Created cluster with these nodes. Cluster creation was successful. Creation of pods was fine. Cluster was fucntional

2. Created DO nodes with docker version 18.09.2. Created cluster with these nodes. Cluster creation was successful. Creation of pods was fine. Cluster was fucntional
